src/devices/Dhtxx/DhtSensor.cs

```diff
@@ -106,7 +80,7 @@ public DhtSensor(int pin, DhtType dhtType, PinNumberingScheme pinNumberingScheme
             _protocol = CommunicationProtocol.OneWire;
             _controller = new GpioController(pinNumberingScheme);
             _pin = pin;
-            _dhtType = dhtType;
+            SetDevice(dhtType);
```

Comment:
Why do we even have a DhtType still, can't we just instead change the constructor to take a IDhtDevice instead? Seems to me that DhtType is not really useful anymore but I may be mistaken.

Reply:
Oh nvm, I just saw that dhtType is simply an enum. I like this new refactoring 👍